test/dynamo/test_higher_order_ops.py

```diff
@@ -2126,6 +2124,410 @@ def wrapper_fn(x, y):
         )
         self.assertEqual(actual, expected)
 
+    def test_vmap(self):
+        def fn(x):
+            return torch.func.vmap(lambda x: x.sum(0) + x.sum(1))(x)
+
+        x = torch.randn(3, 3, 3)
+        wrapped_gm = self._compile_check(fn, (x,))
+
+        # Dynamic shapes produce a slightly different graph.
+        if check_dynamic_shape_capture():
+            return
+
+        expected = """\
+class GraphModule(torch.nn.Module):
+    def forward(self, L_x_ : torch.Tensor):
+        l_x_ = L_x_
+
+        _check_randomness_arg = torch._functorch.vmap._check_randomness_arg('error')
```

Comment:
Where did this come from? This probably shouldn't be in the graph (the only things that go in the graph should be Tensors and operators)

Reply:
concretely the thing I think we/someone should add (in a separate PR) is a way to mark a function as always inlined by Dynamo, even if it exists in a skipfiles

Reply:
Tracked in https://github.com/pytorch/pytorch/issues/100278